packages/@webex/plugin-meetings/src/multistream/mediaRequestManager.ts

```diff
@@ -213,6 +223,10 @@ export class MediaRequestManager {
 
     mediaRequest.receiveSlots.forEach((rs) => {
       rs.on(ReceiveSlotEvents.SourceUpdate, this.sourceUpdateListener);
+      rs.on(ReceiveSlotEvents.MaxFsUpdate, ({maxFs}) => {
+        mediaRequest.preferredMaxFs = maxFs;
```

Comment:
a media request will have a number of receive slots, so here we're just storing the maxFs we got from only one of them (the last one that emitted the event). Shouldn't we store the minimum? Or is this assuming that all receive slots within a group will have same size hints?

Reply:
I'm assuming that all receive slots in a group will have the same size hints. It is a necessary assumption as the alternative of storing the minimum would mean that you would degrade a large video within a group because of the presence of a small video. Basically you can't have a group with different sized panes in it.